app/services/start_accession.rb

```diff
@@ -3,13 +3,14 @@
 # Starts the accession workflow by calling dor-services-app
 # See https://sul-dlss.github.io/dor-services-app/#operation/objects#accession
 class StartAccession
-  def self.run(druid:, user:)
+  def self.run(druid:, user:, workflow: nil)
     object_client = Dor::Services::Client.object(druid)
 
     object_client.accession.start(
       significance: 'major',
       description: 'pre-assembly re-accession',
-      opening_user_name: user
+      opening_user_name: user,
+      workflow:
     )
```

Comment:
Allow us to tell DSA which exact workflow we want started instead of just accepting the default